chan_dongle.h

```diff
@@ -102,6 +102,11 @@ typedef struct pvt_stat
 
 struct at_queue_task;
 
+typedef unsigned int sms_inbox_item_type;
+
+#define SMS_INBOX_ITEM_BITS     (sizeof(sms_inbox_item_type) * 8)
+#define SMS_INBOX_ARRAY_SIZE    ((SMS_INDEX_MAX / SMS_INBOX_ITEM_BITS) + 1)
```

Comment:
`((SMS_INDEX_MAX + SMS_INBOX_ITEM_BITS - 1) / SMS_INBOX_ITEM_BITS) /* (256+32-1)/32 == 8 */`

Not because I care about that lost memory, but because this is self-explanatory, and the addition of `+1` is not.